src/routes/Receive.tsx

```diff
@@ -459,6 +460,16 @@ export function Receive() {
                                 <p>{error()}</p>
                             </InfoBox>
                         </Show>
+                        <Show
+                            when={
+                                flavor() === "onchain" &&
+                                state.federations?.length
```

Comment:
we should just always show this for on-chain. we only sync an address once for our own on-chain wallet so we'll have a similar issue

Reply:
fixed